test/live.test.ts

```diff
@@ -24,6 +27,29 @@ const testSse = async (onRequest: OnRequest, options: ClientConfig = {}) => {
 describe.skipIf(typeof EdgeRuntime === 'string' || typeof document !== 'undefined')(
   '.live.events()',
   () => {
+    const server = setupServer(
```

Comment:
As explained further down, `nock` isn't able to intercept `global.fetch`, so we use `msw` for this.
We might be able to use `msw` instead of `nock`, but it feels out of scope for this PR.